stanford/roles/cluster/defaults/main.yml

```diff
@@ -537,3 +543,46 @@ CLUSTER_SERVICE_TYPE: 'ec2'
 # alternate values are
 # CLUSTER_SERVICE_TYPE: 'rds'
 # CLUSTER_SERVICE_TYPE: 'emr'
+cluster_emr_internal_group_rules: &cluster_emr_internal_group_rules
+  - proto: icmp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][0]['cidr']"
+  - proto: icmp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][1]['cidr']"
+  - proto: tcp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][0]['cidr']"
+  - proto: tcp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][1]['cidr']"
+  - proto: udp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][0]['cidr']"
+  - proto: udp
+    from_port: 0
+    to_port: 65535
+    cidr_ip: "{{ CLUSTER_SUBNETS['emr'][1]['cidr']"
+cluster_emr_internal_group_rules_slave: "{{ cluster_emr_internal_group_rules }}"
+cluster_emr_internal_group_rules_master:
+  - *cluster_emr_internal_group_rules
```

Comment:
I think you don't want the leading hypen `-` here:

If you look at an example from `playbooks/roles/edxapp/defaults/main.yml`:

```yaml
lms_auth_config:
  <<: *edxapp_generic_auth
  SEGMENT_KEY: "{{ EDXAPP_LMS_SEGMENT_KEY }}"
  # etc...
```

I think we want something of the form:

```yaml
cluster_emr_internal_group_rules_master:
  <<: *cluster_emr_internal_group_rules
  - proto: 'tcp'  # etc...
```